packages/replay-next/components/inspector/HTMLChildrenRenderer.tsx

```diff
@@ -33,17 +36,24 @@ export default function HTMLChildrenRenderer({
   return (
     <>
       {childNodes.map((objectId: ProtocolObjectId, index: number) => (
-        <HTMLChildRenderer key={index} objectId={objectId} pauseId={pauseId} />
+        <HTMLChildRenderer
+          key={index}
+          objectId={objectId}
+          path={addPathSegment(path, `${index}`)}
```

Comment:
Calling a function during render is a [side effect](https://reactjs.org/docs/strict-mode.html#detecting-unexpected-side-effects). This will end up being called many times which is probably _not_ what we want.

Should either use memo in `PropertiesRenderer` to pre-compute (and memoize) all paths, _or_ we could move this callback into an effect. (Not sure which makes the most sense as I'm still reading through this PR– but I assume `useMemo`).